cmd/emailreport/README.md

```diff
@@ -0,0 +1,15 @@
+# PnP failure Report tool
+
+This tool reports by email the obtained PnP results from `checkmetrics`
+execution, next it will send the results by email.
+The configuration of SMTP server and checktool arguments are given
+by a configuration file in TOML format, this section includes an
+example of that.
```

Comment:
OK, np - then rather than saying `this section includes`, we could say something like:

        an example TOML configuration file is provided in the file `example.toml`